ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -154,9 +154,9 @@ ompi_mtl_ofi_context_progress(int ctxt_id)
                 ret = ofi_req->event_callback(&ompi_mtl_ofi_wc[i], ofi_req);
                 if (OMPI_SUCCESS != ret) {
                     opal_output(0,
-                                "%s:%d: Error returned by request (type: %d) event callback: %zd.\n"
+                                "%s:%d: Error returned by request event callback: %zd.\n"
```

Comment:
could you cache the req type before invoking the event_callback and use that in the error message?  might help a little with debugging.

Reply:
Ah that's smart. Let me do that...

Reply:
Updated. Running our CI again...